packages/keystone/lib/adapters/index.js

```diff
@@ -159,6 +159,13 @@ class BaseFieldAdapter {
     this.isUnique = isUnique;
   }
 
+  get list() {
+    return this.getListByKey(this.listAdapter.key);
+  }
+  get field() {
+    return this.list.fieldsByPath[this.path];
+  }
```

Comment:
Can we make these `getList()` and `getField()` methods instead of getters? I'm not a fan of getters in general, but `list` and `field` in particular are almost impossible to grep in this codebase without a lot of noise.

Reply:
Hmm.. but this is exactly what getters are good at though and they _meant_ to look like fields from the outside. `this.field.blah` is a lot more natural than `this.getField.blah` don't you think? The latter would confuse me -- looks like a typo, should be `this.getField().blah`.

It's always going to be hard to grep for things with common names, I don't think that makes the names wrong in this case.

Can we get a third opinion? @JedWatson @timleslie

Reply:
> should be `this.getField().blah`

Yes, that's what I meant by _"`getFields()` methods_".

> It's always going to be hard to grep for things with common names, I don't think that makes the names wrong in this case.

It's getting worse and worse in the Keystone codebase. It's pain I'm feeling daily, so want to avoid it at every possible turn. This is one of them.

If I `rg` the codebase for `field` today, there are 3,328 results. `getField()` however; 61 results. Similarly for `list`; 2,742 vs 88.